pysilcam/silcamgui/guicalcs.py

```diff
@@ -295,25 +303,25 @@ def silcview(datadir):
             dirtxt = '<<'
         if pause:
             dirtxt = 'PAUSED ' + dirtxt
-        label = font.render('DIRECTION [<-|->|p]: ' + dirtxt, 1, (255,255,0))
+        label = font.render('DIRECTION [<-|->|p]: ' + dirtxt, 1, font_colour)
         screen.blit(label, (0, size[1]-40))
 
         if counter == 0:
-            label = font.render('FIRST IMAGE', 1, (255,255,0))
+            label = font.render('FIRST IMAGE', 1, font_colour)
             screen.blit(label, (0, size[1]-60))
         elif counter == len(files)-1:
-            label = font.render('LAST IMAGE', 1, (255,255,0))
+            label = font.render('LAST IMAGE', 1, font_colour)
             screen.blit(label, (0, size[1]-60))
 
         timestamp = pd.to_datetime(
                 os.path.splitext(os.path.split(f)[-1])[0][1:])
 
 
         pygame.display.set_caption('raw image replay:' + os.path.split(f)[0])#, icontitle=None)
-        label = font.render(str(timestamp), 20, (255, 255, 0))
+        label = font.render(str(timestamp), 20, font_colour)
         screen.blit(label,(0,0))
         label = font.render('Display FPS:' + str(c.get_fps()),
-                1, (255, 255, 0))
+                1, font_colour)
```

Comment:
E128: continuation line under-indented for visual indent  <!-- Automatically posted by Lintly -->